androdeb

```diff
@@ -53,14 +54,45 @@ case $key in
 esac
 done
 
-if [ ! -z "$PULL" ]; then
+##########################################################
+#  GIT-PULL
+##########################################################
+
+if [ ! -z "$GIT_PULL" ]; then
 	echo "Updating androdeb by git pull"
 	cd $spath
 	git pull
 	echo "Done."
 	exit 0
 fi
 
+##########################################################
+#  PULL
+##########################################################
+if [ ! -z "$PULL" ]; then
+	if [ $1 == "-a" ]; then
+		PRESERVE="-a"
+		echo "Preserving filestamps and mode"
+		shift || true
+	fi
+
+	if [ "2" -le "$#" ]; then
```

Comment:
why not: `if [ $# -le 2 ]`

Reply:
I need to check if 2 or more arguments are remaining (multiple files / local destination specified), I guess it could be:

`if [$# -gt 1 ]`